src/code/sourceLocationTracker.js

```diff
@@ -67,7 +67,7 @@ function getSourceMap (address, code, contracts) {
   var isCreation = helper.isContractCreation(address)
   var byteProp = isCreation ? 'bytecode' : 'runtimeBytecode'
   for (var k in contracts) {
-    if ('0x' + contracts[k][byteProp] === code) {
```

Comment:
A proper fix should strip the constructor arguments from code and then do a full equality comparison. Is that feasible in the current architecture?